### DISCUSSION TITLE
Outlook and Yahoo Calendars not appearing

### DISCUSSION THREAD

**Discussion by dfabro123** [add2cal/add-to-calendar-button#494] [Q&A]
I've created my button but for some reason it isn't showing Outlook and Yahoo options:
![image](https://github.com/add2cal/add-to-calendar-button/assets/149180546/d148e89f-af16-4a16-919d-ed2cb6cbc0ff)

I believe everything in my code is correct but let me know if I'm misguided: 

![image](https://github.com/add2cal/add-to-calendar-button/assets/149180546/f3af6c27-b247-479a-b75d-6beef30c0f9a)

Apart from this, it works for Google Calendar. By the way, are there any options to edit button visuals? 

Thanks

**Comment by jekuer** [comment]
Hi @dfabro123 ,
This is the expected behavior. As described in the documentation at https://add-to-calendar-button.com/configuration#recurrence, recurrence would deactivate Microsoft and Yahoo options, as they are currently not supporting those type of events (at least not via an external trigger).
